mopub/mopub-android-mediation
# Added onClick Support for Tapjoy_Mopub Android Adapter

This PR is regarding onClick callback support for Tapjoy adapter for Mopub android SDK.  onClick callback will fire when a click event has occurred on Tapjoy placement. 

What's new:
- public void onClick(TJPlacement placement) { } is added to TapjoyInterstitial and TapjoyRewardedVideo classes of Tapjoy adapter.

*Notes: `onClick ` callback  won't get fired or be fully functioning currently until Tapjoy server change is completed near future.

How to test:
- Run MoPub sample app with latest Mopub SDK, Tapjoy SDK and Tapjoy Adapter from this branch.
- Request Tapjoy video placement. 
- Show and play the video ads.
- Perform a click on video ads and observe in Logcat if `onClick` callback gets fired on this placement from Tapjoy Network.
